part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+
+Another use-case for using hierarchies might be because you're running a "margin business". That is, you're only buying what you're selling and the cost of goods sold is the large part of your expenses. In cases like this it can be a good idea to use a different setup of the income statement: one where cost of goods sold is immediately subtracted from income, leaving gross margin to cover all other expenses in the company.
+
+In order to set a heading's parent, go to the \menupath{General Journal \ma Chart of Accounts}. Click on the account number. If the account is a heading account that has a parent heading, the resulting screen shown in \figref{fig:config-coa-hierarchy}.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\graphicswidth]{images/gl-coa-heading.png}
+    \caption{Configure Account Hierarchy}
+    \label{fig:config-coa-hierarchy}
+\end{figure}
+
+If the account is a not a heading account, but has a parent heading, then the result is shown in \figref{fig:coa-account-setup}.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{images/gl-coa.png}
+    \caption{Account Setup}
+    \label{fig:coa-account-setup}
+\end{figure}
+
+When creating "Income Statement" \index{income statement} or "Balance Sheet" \index{balance sheet}reports, for example, \menupath{Reports \ma Income Statement} you can select to report using the full hierarchy or just the account category.  See \figref{fig:config-report-hierarchy}
+
+\begin{figure}[H]
+\centering
+\includegraphics[width=\graphicswidth]{images/reports-select-hierarchy.png}
+\caption{Report Using Hierarchy}
+\label{fig:config-report-hierarchy}
+\end{figure}
+
+\subsection{Account setup}
+\label{sec-coa-account-setup}
+
+Headers don't have any configuration, other than their number and description. 
+
+Accounts also
+have a number and description, but require additional configuration for the application to work
+correctly. 
+
+The settings, which are shown in \figref{fig:coa-account-setup} are described in the following paragraphs.
+
+The main setup consists of the following:
+
+\begin{description}[style=nextline]
+    \item[Account Number] Income statements and balance sheets reports are ordered by account number. Traditionally, account numbers are 4 digits with accounts being grouped by the first digit, but this type of grouping is not enforced by LedgerSMB. Grouping is handled using headings as explained in \secref{sec-coa-accounts-structure}.
+    \item[Description] Should be a relative short, easy to understand description of what the account is used for.
+    \item[Account Type] Asset, Liability, Equity, Equity temporary, Income or Expense.
+    \item[Heading] Optional entry. Used if this account is a child of a heading account.
+    \item[Negative balance heading] @@@ TODO

```

Comment:
Balance sheet account balances should - generally - not be reported as negative amounts[^1]: a negative asset is in fact a liability. This functionality reports the given account under the primary header as long as the asset/liability is positive and reports it under the "negative balance heading" as soon as the asset/liability turns negative (the expectation is that the alternative header is a liability/asset header instead of an asset/liability header).

[^1]: Notable exception here are "cumulative depreciation" accounts in fixed asset accounting, but these are usually marked as "Contra" -- causing them to be reported as positive numbers...

Reply:
Added verbatim.